### ISSUE TITLE
tensorflow.keras model.fit_generator not working with use multiprocessing

### ISSUE THREAD

**Issue by thebeancounter** [tensorflow/tensorflow#27260]
I am using Ubuntu 18 with conda and python 3.6 with tf 1.20 GPU over a gtx 1070 and 2080ti

When I am using this code 

    model.fit_generator(i, epochs=1, workers=16,
                            use_multiprocessing=False, max_queue_size=16,
                            verbose=1)

It works fine

When I change use_multiprocessing to True, and run, it does nothing, the cpu is not utilized by python at all, and nothing shows on the monitor. It used to work fine in the past. 

What am I missing here?

**Comment by ymodak** [comment]
@thebeancounter Since you mentioned that ```use_multiprocessing``` was working for you in the past. So did you recently updated your configuration or updated the code that caused change in the behavior? Can you please add a little bit context here? Thanks!

**Comment by thebeancounter** [comment]
@ymodak 
I am afraid I do not have the record of the working code and system settings and was not able to reproduce a working environment.

**Comment by ymodak** [comment]
Okay. I see you have mentioned that you are using TF 1.20. Is there a possibility for you to switch to latest version of TF and try again?

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by cobujo** [comment]
I was getting an error trying to use fit_generator and one of the strategies in tf.distribute.Strategy and came across this thread.  So it looks like use_multiprocessing as well as any tf strategies that use multiple workers/GPUs does not work with fit_generator.

What would be the best solution here?  Do I have to change my generator to a tf.Dataset object?

**Comment by tsing-cv** [comment]
@thebeancounter have you solved this problem?

**Comment by ashkan-mokarian** [comment]
I am facing the same issue here, was using tf.keras use_multiprocessing=True in my fit_generator function without any issues, came back few month later and now nothing runs, exactly as @thebeancounter mentioned. only difference is that I used to use TF 1.12 together with Cuda9.2 and also the corresponding drivers. Now I have only updated TF to 1.14 together with Cuda10.1, nothing else has changed.
